src/Microsoft.AspNetCore.Mvc.Core/breakingchanges.netcore.json

```diff
@@ -32,5 +32,129 @@
   {
     "TypeId": "public class Microsoft.AspNetCore.Mvc.ModelBinding.JQueryFormValueProvider : Microsoft.AspNetCore.Mvc.ModelBinding.BindingSourceValueProvider, Microsoft.AspNetCore.Mvc.ModelBinding.IEnumerableValueProvider",
     "Kind": "Removal"
+  },
+  {
+    "TypeId": "public class Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ArrayModelBinderProvider : Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinderProvider",
```

Comment:
@javiercn / @dougbu / @rynowak - what would this breaking change be?

I see it in 2.0: https://github.com/aspnet/Mvc/blob/2.0.3/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ArrayModelBinderProvider.cs
And in dev: https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ArrayModelBinderProvider.cs

Reply:
I can't see `ArrayModelBinderProvider` in the "before" baseline and don't see any changes for this type since 2.0. I confirmed the "before" gap using `git diff origin/release/2.1..origin/rybrande/baselines`. Something very strange is going on.

I also can't understand how the new baselines, which contain exactly these `TypeId` values, can lead to the build errors @ryanbrandenburg mentioned to me offline. Suspect we need to debug API Check and see what it's generating for the current assembly.

Reply:
Fixed a typo in last comment: "**should** contain"

Reply:
Yes, best I can tell is this is another instance of aspnet/BuildTools#615. With your blessing I'll file the issue to track reverting it.

Reply:
Filed #7521.